dynawo/sources/Models/CPP/Controls/Voltage/ModelVoltageMeasurementsUtilities/DYNModelVoltageMeasurementsUtilities.cpp

```diff
@@ -199,6 +202,8 @@ ModelVoltageMeasurementsUtilities::getIndexesOfVariablesUsedForCalculatedVarI(un
     case minValIdx_:
     case maxValIdx_:
     case avgValIdx_:
+    case minIValIdx_:
+    case maxIValIdx_:
```

Comment:
Not coherent with the evalJ above

Reply:
It does seem like this, but this might be an artefact of what is being computed. Namely the index of the voltage level achieving the min (or max). 
From a more mathematical point of view, any change in any one of the voltage values indicated through the push_back here could, in theory change the value of the variable min/max and hence change the index achieving this value. However, they all have a 0 derivative strictly speaking. 

From a computational point of view, I agree, since the Jacobian is in fact always full of 0s one may say that the variables aren't impacted by any of the variables mentioned in this getIndexes... function. 

Probably my misunderstanding comes from when this function is called and what it triggers: if it is only to tell when to recompute the Jacobian, then yes, this function needs changes. If it to tell the algorithm when to re-evaluate the actual variable (min index / max index), then the way it is written now is, I believe, correct.